shell/app/styles/antd-extension.scss

```diff
@@ -414,6 +414,16 @@ div.ant-table {
   }
 }
 
+.ant-form-item-required:not(.ant-form-item-required-mark-optional)::before {
```

Comment:
in antd 3.x
![image](https://user-images.githubusercontent.com/5175455/126959906-70b86777-1337-4dc8-9860-45bf6e38e4c9.png)
in antd 4.x
![image](https://user-images.githubusercontent.com/5175455/126959989-a3f91c6e-845e-4844-b411-36ae3eb09011.png)

from the snapshot, u can see, `ant-form-item-required` is a global antd 3.x class, but in 4.x it has one more class selector, that't the reason why we lost it.

OK. first thing is antd-extension.scss is used to override antd default style. but in your case, u didn't override any style. Instead u're creating a new style class, from my perspective, has not relation with antd. So remove it from here.

second thing is how to get it back, two solutions, 1. add a new class in app.scss 2. use tailwind to implement it. Better try the second way